specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/workspace.json

```diff
@@ -1504,7 +1504,13 @@
         },
         "azureADOnlyAuthentication": {
           "description": "Enable or Disable AzureADOnlyAuthentication on All Workspace subresource",
-          "type": "boolean"
+          "type": "boolean",
+          "x-ms-mutability": [
+            "create",
+            "update",
+            "read"
+          ]
+
```

Comment:
This change is unnecessary as it has exactly the same behavior as without x-ms-mutability.  See:

https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-mutability